pom.xml

```diff
@@ -473,7 +474,6 @@ THE SOFTWARE.
                   <module name="CovariantEquals" />
                   <module name="EqualsHashCode" />
                   <module name="OneStatementPerLine" />
-                  <module name="PackageDeclaration" />
```

Comment:
There weren't any violations of this rule in main sources, but there were some violations in test sources. Looking at the rule, it didn't seem to provide much value in the first place, so I chose to disable it for now. We can always enable it again in the future if we feel that the value it provides is worth fixing the violations in test sources.